apps/verify/behaviours/send-verification-email.js

```diff
@@ -73,13 +73,15 @@ module.exports = superclass => class extends superclass {
     const unSubmittedCase = _.filter(response.data, record => !record.submitted_at);
     const unSubmittedCaseEmail = unSubmittedCase.map(record => { return record.email; });
 
+
     // if form has not been submitted, throws an error if a second form is opened with same UAN but different email
+
     if (recordEmail.length && req.form.values['user-email'] !== unSubmittedCaseEmail.toString() && unSubmittedCase.length > 0) {
       return next({
         'user-email': new this.ValidationError(
           'user-email',
           {
-            type: 'noRecordMatch'
+            type: 'emailNotExist'
```

Comment:
This error happens when the current case email does not match the email entered (as the email for a DB entry cannot be reassigned). I think this error should continue to give the `noRecordMatch` validation error message.

Reply:
Yes but the ucd don't want the displayed error message rather then the one on the ticket i believe

Reply:
I believe it's the 500 error you get when you input an email that doesn't exist that needs to be replaced with the validation error, as you have done.

However we still need a specific error for the case where a user tries to continue a referral with the wrong email address.

Maybe check with UCD?